examples/config-sync.json

```diff
@@ -0,0 +1,52 @@
+{
+	"version":"0.1.0-dev",
+	"storage":{
+		"rootDirectory":"/tmp/zot"
+	},
+	"http":{
+		"address":"127.0.0.1",
+		"port":"5000"
+	},
+	"log":{
+		"level":"debug"
+	},
+	"extensions":{
+		"sync":[
+			{
+				"url": "https://registry1:5000",
+				"pollInterval": "6h",
+				"tlsVerify": true,
+				"credentials": {
```

Comment:
Would consider "credentials" to point to a separate file instead, so that we can align with kubernetes secrets [1] which can be mounted as a file also.

[1] https://kubernetes.io/docs/concepts/configuration/secret/

Reply:
Ok, added credentials file along with plain text